k9mail/src/main/java/com/fsck/k9/planck/infrastructure/modules/ViewModelModule.kt

```diff
@@ -17,7 +17,7 @@ import dagger.hilt.android.qualifiers.ApplicationContext
 @Module
 @InstallIn(ActivityRetainedComponent::class)
 class ViewModelModule {
```

Comment:
Do we still need this Module if it's fully commented out?

Reply:
omg I forgot to remove it, nice catch.